### ISSUE TITLE
Not Working With php 7 

### ISSUE THREAD

**Issue by burhandodhy** [AtomLinter/linter-php#145]
linter-php: Unable to determine PHP version
Unable to determine the version of "C:\xampp\php", please verify that this is the right path to PHP.

**Comment by Arcanemagus** [comment]
What does `php -v` output on your system? It's working with PHP 7 in CI builds and on my machines...

**Comment by burhandodhy** [comment]
PHP 7.0.3 (cli) (built: Feb  2 2016 14:41:12) ( ZTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies

**Comment by Arcanemagus** [comment]
That definitely works: https://regex101.com/r/xF8vC4/1

Can you open Developer Tools, put a breakpoint [here](https://github.com/AtomLinter/linter-php/blob/v1.2.0/lib/main.coffee#L18), and then in Atom disable and then enable `linter-php`?

This should trigger the breakpoint, once it triggers can you post the result of running the following in the console:

```js
JSON.stringify(output)
regex.exec(output)
```

**Comment by burhandodhy** [comment]
I delete linter and linter-php  package , restart my pc and again install both packages again , now it working perfectly

**Comment by Arcanemagus** [comment]
How bizarre, at least it's working for you now. If you have any further problems feel free to file an issue :wink:.
